go/worker/merge/committee/node.go

```diff
@@ -356,11 +356,10 @@ func (n *Node) tryFinalizeResultsLocked(pool *commitment.Pool, didTimeout bool)
 	}()
 
 	epoch := n.commonNode.Group.GetEpochSnapshot()
-	// NOTE: The roothash backend will start counting its timeout on its own based on
-	//       any received commits so in the worst case the actual timeout will be
-	//       2*roundTimeout.
-	params := epoch.GetRoothashParams()
```

Comment:
Do we still need `GetRoothashParams` after this change? Probably not, so it is best to just remove it (and anything related that is not needed anymore).